ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/SandboxPartyManagementService.scala

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.services.admin
+
+import akka.stream.Materializer
+import com.daml.ledger.participant.state.index.v2.{
+  PartyManagementService => StatePartyManagementService
```

Comment:
Hasn't Gabor mentioned that these constituent interfaces of the Index service will ultimately be prefixed by `Index`? If yes, consider using that convention already

Reply:
Renamed in https://github.com/digital-asset/daml/pull/1452/commits/4691df2c69cc34bf11a712f83de124b567575b3c